core/src/test/java/org/bitcoinj/evolution/ProviderTests.java

```diff
@@ -70,6 +70,7 @@ public class ProviderTests {
 
     @Before
     public void startup() throws UnreadableWalletException {
+        BLSScheme.setLegacyDefault(true);
```

Comment:
with Core 19, there is the introduction of Legacy and Basic Schemes for BLS keys.

Many of the previous tests were written using the Legacy Scheme, so we have to add this to some tests to make sure they pass.